### PR TITLE
feat(server-routes): implement persisting input values in localStorage

### PR DESCRIPTION
**Pull request by arashsheyda**
https://github.com/nuxt/devtools/assets/38922203/b7391def-242e-4de6-9a09-14eaa2b33c7c

### CODE REVIEW

**Comment by Atinux** [comment]
Shall we also remember the active tab for each route?

**Comment by Atinux** [comment]
Looks good to me! I let @antfu do the final review

File: packages/devtools/client/components/ServerRouteDetails.vue [hunk 1]
```diff
@@ -308,6 +308,50 @@ watchEffect(() => {
   }
 })
 
+const savedRouteInputs = useLocalStorage<{ path: string, tab: string, inputs: any }[]>('nuxt-devtools:server-routes:inputs', () => [], {
+  window: window.parent,
```
Comment (antfu) [inline packages/devtools/client/components/ServerRouteDetails.vue:312]: Is there a reason to pass the parent window here? It should work with iframe window as they are same origin.

Comment (arashsheyda) [reply packages/devtools/client/components/ServerRouteDetails.vue:312]: While trying out with my laptop(windows 10), the storage wasn't working as expected(the client would get reset after a page refresh)
Is it possible for you to test it out?

**Pull request merged**